### DISCUSSION TITLE
In version 10, the functionality of the left menu is gone.

### DISCUSSION THREAD

**Discussion by 6LWa6ZKx** [apache/skywalking#12283] [Q&A] [answered]
I just got to know this library and my installation is 10.0 and UI. You can see the service while it is running.

But unlike other tutorials on the Web, the left side is missing a lot of menu functionality.

For example, the "browser, gateway" series. How do I open it?

![_20240531192648](https://github.com/apache/skywalking/assets/61306859/6648bcf6-0c45-4f4d-990d-e0f34508271c)

**Comment by CodePrometheus** [comment]
The corresponding UI will be displayed when there is data available.

**Comment by wu-sheng** [comment, marked as answer]
Once services of that layer are discovered from telemetry data, the menu will show up automatically.

This is not a v10 feature at all.

**Reply by 6LWa6ZKx** [reply]
> 一旦从遥测数据中发现该层的服务，菜单就会自动显示。
> 
> 这根本不是 v10 的功能。

Thank you for your answers。

I installed nginx-lua, which hasn't been updated in a while.

In the following figure, I annotate the actual execution order, but the traceId generated by nginx seems to be not passed correctly, and the node "User" is redundant.
My "socket" also comes from the "view" node. I think the Vue project "view" node is the beginning of everything.

Did I do something wrong?

![_20240603085819](https://github.com/apache/skywalking/assets/61306859/9e167180-8ec0-4523-b085-ffd4322926e6)

**Reply by wu-sheng** [reply]
nginx-lua is only doing tracing, which has nothing about nginx server monitoring. Ngninx monitoring is https://skywalking.apache.org/docs/main/latest/en/setup/backend/backend-nginx-monitoring/

About what you ask, I am not following actually. Because what is on the topology is determined by two things
1. Time Range. 
2. Generated traces.

Meanwhile, TCP is not traceable, as there is no header to propagate, this is one basic tracing knowledge. You should learn how the trace continuous cross process.

You seems to mix different topics in one discussion.

**Reply by 6LWa6ZKx** [reply]
> nginx-lua is only doing tracing, which has nothing about nginx server monitoring. Ngninx monitoring is https://skywalking.apache.org/docs/main/latest/en/setup/backend/backend-nginx-monitoring/
> 
> About what you ask, I am not following actually. Because what is on the topology is determined by two things
> 
> 1. Time Range.
> 2. Generated traces.
> 
> Meanwhile, TCP is not traceable, as there is no header to propagate, this is one basic tracing knowledge. You should learn how the trace continuous cross process.
> 
> You seems to mix different topics in one discussion.

Well, thank you for your answer.

I will continue to study.
